go.mod

```diff
@@ -0,0 +1,130 @@
+module github.com/operator-framework/operator-sdk
+
+go 1.12
+
+require (
+	github.com/BurntSushi/toml v0.3.1
+	github.com/coreos/go-semver v0.2.0
+	github.com/coreos/prometheus-operator v0.29.0
+	github.com/ghodss/yaml v1.0.0
+	github.com/go-logr/logr v0.1.0
+	github.com/go-logr/zapr v0.1.1
+	github.com/iancoleman/strcase v0.0.0-20180726023541-3605ed457bf7
+	github.com/markbates/inflect v1.0.4
+	github.com/martinlindhe/base36 v0.0.0-20180729042928-5cda0030da17
+	github.com/mattbaird/jsonpatch v0.0.0-20171005235357-81af80346b1a
+	github.com/mitchellh/go-homedir v1.1.0
+	github.com/operator-framework/operator-lifecycle-manager v0.0.0-20190128024246-5eb7ae5bdb7a
+	github.com/pborman/uuid v1.2.0
+	github.com/pkg/errors v0.8.1
+	github.com/prometheus/client_golang v0.9.3-0.20190127221311-3c4408c8b829
+	github.com/prometheus/client_model v0.0.0-20190129233127-fd36f4220a90
+	github.com/prometheus/common v0.2.0
+	github.com/prometheus/prometheus v0.0.0-20190328180107-4d60eb36dcbe
+	github.com/rogpeppe/go-internal v1.3.0
+	github.com/sergi/go-diff v1.0.0
+	github.com/sirupsen/logrus v1.4.1
+	github.com/spf13/afero v1.2.2
+	github.com/spf13/cobra v0.0.3
+	github.com/spf13/pflag v1.0.3
+	github.com/spf13/viper v1.3.2
+	github.com/stretchr/testify v1.3.0
+	go.uber.org/zap v1.9.1
+	golang.org/x/tools v0.0.0-20190408170212-12dd9f86f350
+	gopkg.in/yaml.v2 v2.2.2
+	k8s.io/api v0.0.0-20190612125737-db0771252981
+	k8s.io/apiextensions-apiserver v0.0.0-20190228180357-d002e88f6236
+	k8s.io/apimachinery v0.0.0-20190612125636-6a5db36e93ad
+	k8s.io/cli-runtime v0.0.0-20181213153952-835b10687cb6
+	k8s.io/client-go v11.0.0+incompatible
+	k8s.io/code-generator v0.0.0-20181203235156-f8cba74510f3
+	k8s.io/gengo v0.0.0-20190327210449-e17681d19d3a
+	k8s.io/helm v2.13.1+incompatible
+	k8s.io/klog v0.3.1
+	k8s.io/kube-openapi v0.0.0-20190320154901-5e45bb682580
+	k8s.io/kube-state-metrics v1.6.0
+	k8s.io/kubernetes v1.11.8-beta.0.0.20190124204751-3a10094374f2
+	sigs.k8s.io/controller-runtime v0.1.10
+	sigs.k8s.io/controller-tools v0.0.0-20190411181648-9d55346c2bde
+)
+
+// Pinned to kubernetes-1.13.1
+replace (
+	k8s.io/api => k8s.io/api v0.0.0-20181213150558-05914d821849
+	k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.0.0-20181213153335-0fe22c71c476
+	k8s.io/apimachinery => k8s.io/apimachinery v0.0.0-20181127025237-2b1284ed4c93
+	k8s.io/apiserver => k8s.io/apiserver v0.0.0-20181213151703-3ccfe8365421
+	k8s.io/cli-runtime => k8s.io/cli-runtime v0.0.0-20181213153952-835b10687cb6
+	k8s.io/client-go => k8s.io/client-go v0.0.0-20181213151034-8d9ed539ba31
+	k8s.io/code-generator => k8s.io/code-generator v0.0.0-20181117043124-c2090bec4d9b
+	k8s.io/kube-aggregator => k8s.io/kube-aggregator v0.0.0-20181213152105-1e8cd453c474
+	k8s.io/kube-openapi => k8s.io/kube-openapi v0.0.0-20180711000925-0cf8f7e6ed1d
+	k8s.io/kubernetes => k8s.io/kubernetes v0.0.0-20190201210629-c6d339953bd4
+)
+
+require (
```

Comment:
Any reason there are two `require` blocks?

Reply:
I think one is for indirect dependencies 😄

Reply:
> I think one is for indirect dependencies

Yep, that's the reason. I figured splitting them up may make it easier to manage our direct dependencies. The downside is that the go toolchain isn't smart enough to separate them like this on it's own, so whenever we add or update a dependency, there will likely be some manual editing to move some new indirect dependencies down into the separate group.

I can put them all together again if we'd rather.

Reply:
I haven't seen other projects use this organization, and it adds a bit of extra overhead as you mentioned. I vote to keep them all in one block.